qcodes/instrument_drivers/Keysight/N51x1.py

```diff
@@ -42,7 +43,7 @@ def __init__(self, name, address, **kwargs):
         self.add_parameter('rf_output',
                            get_cmd='OUTP:STAT?',
                            set_cmd='OUTP:STAT ' + '{:s}',
-                           val_mapping={'on': 1, 'off': 0})
```

Comment:
I don't understand the need for this change. The number is automatically converted into a str when you generate the string

Reply:
Imho it should just be `set_cmd='OUTP:STAT {}'`

Reply:
indeed, this is what i've seen in the code for the drivers that I've used so far.

Reply:
Yeah that's a better solution
My python 3.6 install throws an exception on the original version for some reason.